ceph.spec.in

```diff
@@ -374,13 +373,16 @@ BuildRequires:	python%{python3_pkgversion}-pyOpenSSL
 BuildRequires:	libxmlsec1-1
 BuildRequires:	libxmlsec1-nss1
 BuildRequires:	libxmlsec1-openssl1
+BuildRequires:	ninja
 BuildRequires:	python%{python3_pkgversion}-CherryPy
 BuildRequires:	python%{python3_pkgversion}-PyJWT
 BuildRequires:	python%{python3_pkgversion}-Routes
 BuildRequires:	python%{python3_pkgversion}-Werkzeug
 BuildRequires:	python%{python3_pkgversion}-numpy-devel
 BuildRequires:	xmlsec1-devel
 BuildRequires:	xmlsec1-openssl-devel
+%else
+BuildRequires:	ninja-build
```

Comment:
Put the line `BuildRequires:  ninja-build` into the preceding `%if 0%{?fedora} || 0%{?rhel}` conditional block.

Reply:
why? what about Ubuntu/Debian and others?

Reply:
ah... it's rpm...

Reply:
> why?

Because we want to be explicit. "Else" is not explicit. Introducing unnecessary "else" blocks into confusing code makes it even more confusing.

> what about Ubuntu/Debian? 

This file is for creating RPM packages. Ubuntu/Debian do not use this file at all.

> and others?

There are no others.

Reply:
@smithfarm thanks, I've updated